CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## 3.1.0 (June 30, 2020)
+
+FEATURES:
+* Storage can be increased its speed via UpdateStorag.
+* Support custom HTTP headers.
+* Be able to deal with GS rate limit.
```

Comment:
Support gridscale API rate limit or bypass gridscale API rate limit??

Reply:
bypass is the one.

Reply:
There's a typo `UpdateStorag.` And maybe change wording here `Be able to deal with GS rate limit.` to `Be able to deal with gridscale API rate limiting.`

Reply:
Done.